dev.bsnes.bsnes.yaml

```
@@ -12,6 +12,7 @@ finish-args:
   - --socket=x11
   - --device=all
   - --filesystem=home:ro
+  - --filesystem=/run/media/mmcblk0p1
```

I don't think `bsnes` will ever reach a state where restricted sandboxing rules really make sense. Most traditional emulators use these set of restrictions. Let's align to them.

```suggestion
  - --filesystem=home
  - --filesystem=/media
  - --filesystem=/run/media
```